.ado/apple-pr.yml

```diff
@@ -10,7 +10,8 @@ pr:
   branches:
     include:
       - master
-      - 0.59-stable
+      - 0.62-stable
```

Comment:
Really this should probably be 

  - master
  - "*-stable"

That way it doesn't need to be updated when additional stable branches are created.

There is no sense in removing old ones, just in case we need to hot fix some really old version for some reason. (Though 0.59 seems pretty unlikely at this point)